jcl/src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -1206,7 +1206,9 @@ public synchronized void start() {
  *
  * @deprecated
  */
-/*[IF Sidecar19-SE]*/
+/*[IF JAVA_SPEC_VERSION >= 18]*/
+@Deprecated(forRemoval=true, since="1.2")
+/*[ELSEIF JAVA_SPEC_VERSION >= 11]*/
```

Comment:
This should be `JAVA_SPEC_VERSION >= 9` to more closely match the meaning of `Sidecar19-SE`.

Reply:
Since 9 is obsolete, not supported, and can't build, using 11 is a good replacement.

Reply:
While I agree it will make no difference in any anticipated future build, writing `JAVA_SPEC_VERSION >= 9` (or if you prefer `JAVA_SPEC_VERSION > 8`) respects the _fact_ that `@Deprecated` acquired the attributes in Java 9. Pretending the boundary occurred at Java 11 just misleads future readers.

Reply:
I expect nobody cares except you, but I've made the update.

Reply:
Perhaps, but there's no cost to being precise.